tf_files/aws/modules/rds/variables.tf

```diff
@@ -0,0 +1,278 @@
+
+variable "rds_instance_create" {
+  description = "Whether to create this resource or not?"
+#  type        = bool
+  default     = true
+}
+
+variable "rds_instance_allocated_storage" {
+  description = "The allocated storage in gibibytes"
+#  type        = "number"
+#  default     = 20
+}
+
+variable "rds_instance_storage_type" {
+  description = "gp2, io1, standard"
+  default     = "gp2"
+}
+
+variable "rds_instance_engine" {
+  description = "The database engine to use"
+#  default     = ""
+}
+
+variable "rds_instance_engine_version" {
+  description = "The engine version to use. If auto_minor_version_upgrade is enabled, you can provide a prefix of the version such as 5.7 (for 5.7.10) and this attribute will ignore differences in the patch version automatically (e.g. 5.7.17)"
+#  default     = ""
+}
+
+variable "rds_instance_instance_class" {
+  description = "The instance type of the RDS instance"
+  default     = "db.t2.micro"
+}
+
+variable "rds_instance_name" {
+  description = "Name for the database to be created"
+  default     = ""
+}
+
+variable "rds_instance_identifier" {
+  description = "The name of the RDS instance, if omitted, Terraform will assign a random, unique identifier"
+  type        = "string"
+}
+
+variable "rds_instance_username" {
+  description = "Username to use"
+#  default     = ""
+}
+
+variable "rds_instance_password" {
```

Comment:
Can we use this instead of a variable?  https://www.terraform.io/docs/providers/random/r/password.html

Also take a look at this: https://www.terraform.io/docs/providers/aws/r/secretsmanager_secret_version.html

Reply:
Random added